senderscore.py

```diff
@@ -3,25 +3,47 @@
 
 
 class SenderScore(BotPlugin):
-    """Plugin to show the mail score of the given ip"""
+    """Plugin to show the mail score of the given adress"""
 
     @botcmd
     def senderscore(self, msg, args):
```

Comment:
because this is a extending for errbot, where args contains the arguments of the message
so its more clear what the args contains / come from